selfdrive/manager/manager.py

```diff
@@ -44,6 +45,7 @@ def manager_init() -> None:
     ("HasAcceptedTerms", "0"),
     ("LanguageSetting", "main_en"),
     ("OpenpilotEnabledToggle", "1"),
+    ("LongitudinalPersonality", str(log.LongitudinalPersonality.standard)),
```

Comment:
```suggestion
    ("LongitudinalPersonality", log.LongitudinalPersonality.standard.real),
```

Reply:
Lol I was looking for this for the longest time but just gave up

Reply:
nvm this doesn't work, can't be int